.github/workflows/pipeline.yml

```diff
@@ -11,19 +11,21 @@ on:
 
 jobs:
   # old-school build and jar method. No tests run or compiled.
-  build-1_6: 
+  build-11: 
```

Comment:
This whole pipeline file needs a lot of cleanup. The entire old "build-1_6" (original lines 14 through 39) section should just be deleted.

Keep just the "Build" section